test/lib/ufe/testComboCmd.py

```diff
@@ -395,3 +414,122 @@ def ball35Scale():
 
         self.runTestCombo(expectedTRS)
 
+    def testRotateScalePivotCompensation(self):
```

Comment:
New test to validate rotate and scale pivot compensation, if the Transform3d interface capabilities support it.